Generate Leads from LinkedIN posts

calok64 opened the pull request:
Branch - Origin/linkedin cert to be pushed to linkedin folder.

Addtional functionality for Create Hubspot tasks from potential leads identified will be added.

jravenel commented:
Hey @calok64 as discussed last week, can you just put more details on the sub-steps of the notebook?

"Apply necessary filters to initial dataframe" and the others are a bit too generic.
Thanks a lot.

calok64 commented:
Hi @jravenel sure will make the changes today and push the latest file

FlorentLvr commented:
@calok64, Hope your doing well !
I just added the HubSpot function in your notebook, rework the structure a bit and made a dynamic calc for your lead classification. 
Could you please review it and make sure it works on your side ? Then, I will merge your PR :)
TY

calok64 commented:
@FlorentLvr i am checking the file. Looks like there are some issues in create task method. working on the same

calok64 commented:
@FlorentLvr Have checked the file and made some required changes.

Key Improvements done :
1) Added logic for building email template
2) Changed the logic of sending email for new contacts created. Instead of sending mail for every contact, it will now include all contacts in one mail.

<img width="679" alt="Capture" src="https://user-images.githubusercontent.com/36129343/157810339-103ab3d6-d435-4203-ad4e-58355f01bc34.PNG">

FlorentLvr commented:
@calok64 ! Well done ! I will merge your PR. :) Thanks for your contribution.
